src/main/java/com/blog/web/repository/UserRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.blog.web.repository;
+
+import com.blog.web.models.UserEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface UserRepository extends JpaRepository<UserEntity, Long> {
+    UserEntity findByEmail(String email);
```

Comment:
usually for finds you want to return an optional it allows for you to check for existence without passing nulls